content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,62 @@
+{{< /tip >}}
+
+## v1alpha1 security policy is not supported anymore
+
+Istio 1.6 no longer supports the [`v1alpha1` authentication policy](https://archive.istio.io/v1.4/docs/reference/config/security/istio.authentication.v1alpha1/) and [`v1alpha1` RBAC policy](https://archive.istio.io/v1.4/docs/reference/config/security/istio.rbac.v1alpha1/), these `v1alpha1` APIs will be ignored starting 1.6.
+
+Instead, Istio 1.6 has split the [`v1alpha1` authentication policy](https://archive.istio.io/v1.4/docs/reference/config/security/istio.authentication.v1alpha1/) and [`v1alpha1` RBAC policy](https://archive.istio.io/v1.4/docs/reference/config/security/istio.rbac.v1alpha1/) into the [`v1beta1` Request Authentication Policy](/docs/reference/config/security/request_authentication) and the [`v1beta1` Peer Authentication Policy](/docs/reference/config/security/peer_authentication). The [`v1alpha1` RBAC policy](https://archive.istio.io/v1.4/docs/reference/config/security/istio.rbac.v1alpha1/) has been replaced by the [`v1beta1` Authorization Policy](/docs/reference/config/security/authorization-policy/). Please migrate to these new resources before upgrading.
+
+To check if there is any `v1alpha1` security policy in the cluster, run the following commands:
+
+{{< text bash >}}
+$ kubectl get policies.authentication.istio.io --all-namespaces
+$ kubectl get meshpolicies.authentication.istio.io --all-namespaces
+$ kubectl get rbacconfigs.rbac.istio.io --all-namespaces
+$ kubectl get clusterrbacconfigs.rbac.istio.io --all-namespaces
+$ kubectl get serviceroles.rbac.istio.io --all-namespaces
+$ kubectl get servicerolebindings.rbac.istio.io --all-namespaces
+{{< /text >}}
+
+To make sure not to apply any `v1alpha1` security policy accidentally in the future, remove the CRD of the `v1alpha1` security policy with the following commands:
+
+{{< text bash >}}
+$ kubectl delete crd policies.authentication.istio.io
+$ kubectl delete crd meshpolicies.authentication.istio.io
+$ kubectl delete crd rbacconfigs.rbac.istio.io
+$ kubectl delete crd clusterrbacconfigs.rbac.istio.io
+$ kubectl delete crd serviceroles.rbac.istio.io
+$ kubectl delete crd servicerolebindings.rbac.istio.io
+{{< /text >}}
+
+# Istio configuration during installation
+
+Historically, Istio has deployed certain configuration objects as part of the installation. This has caused problems with upgrades, confusing user experience, and makes the installation less flexible. As a result, we have minimized the configurations we ship as part of the installation.
+
+This includes a variety of different configurations:
+
+* The `global.mtls.enabled` previously enabled strict mTLS. This should instead be done by directly configuring a `PeerAuthentication` policy for [strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
```

Comment:
```suggestion
* The `global.mtls.enabled` previously enabled strict mTLS. This config is removed to avoid confusion.  You should be configuring a `PeerAuthentication` policy for [strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
```

Reply:
```suggestion
* The `global.mtls.enabled` configuration option previously enabled strict mTLS. This config is removed to avoid confusion.  You should instead configure a `PeerAuthentication` policy for [strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
```